easybuild/easyconfigs/a/ABINIT/ABINIT-7.4.3-goolf-1.4.10.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+#
+##
+
+name = 'ABINIT'
+version = '7.4.3'
+
+homepage = 'http://www.abinit.org/'
+description = """Abinit is a plane wave pseudopotential code for doing condensed phase electronic structure calculations using DFT."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+sources = ['%s-%s.tar.gz' % (name.lower(), version)]
+source_urls = ['http://ftp.abinit.org/']
+
+configopts = '--enable-mpi --enable-mpi-io --with-mpi-prefix=$EBROOTOPENMPI --enable-fallbacks '
+configopts += '--with-netcdf-incs="-I$EBROOTNETCDF/include" --with-netcdf-libs="-L$EBROOTNETCDF/lib -lnetcdf -lnetcdff" '
+configopts += '--with-fft-libs="-L$EBROOTFFTW/lib -lfftw3 -lfftw3f" --with-fft-flavor=fftw3 '
+configopts += '--with-trio-flavor="netcdf+etsf_io" --enable-gw-dpc'
```

Comment:
should be checked against #1551, probably needs a versionsuffix because of the inclusion of ETSF as a dep?